src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveClaimWithDocumentManagementTest.java

```diff
@@ -157,10 +157,13 @@ public void shouldLinkSealedCopyOfRepresentedClaimAfterUpload() throws Exception
             "claim-form.pdf");
     }
 
-    private void assertDocumentIsLinked(ClaimData claimData,
-                                        String authorization,
-                                        ClaimDocumentType claimDocumentType,
-                                        String fileName) throws Exception {
+    private void assertDocumentIsLinked(
```

Comment:
if you are not changing anyting dont format

Reply:
this change was from other PR